crypto/x509/x509_lu.c

```diff
@@ -44,6 +44,13 @@ int X509_STORE_lock(X509_STORE *xs)
     return CRYPTO_THREAD_write_lock(xs->lock);
 }
 
+static int x509_store_lock_ex(X509_STORE *xs, int read)
```

Comment:
Why not just make this a read lock function?
It's called from exactly one place with read==1

Reply:
Done

Reply:
You didn't seem to push?